### ISSUE TITLE
only checkbox clickable option for checkbox and radio filters - jet smart filters.

### ISSUE THREAD

**Issue by 79ho3ein** [CrocoBlock/suggestions#6188]
**Please, describe the feature request, making it clear to understand.**

Hi crocoblock team!

In jet smart filters, if we add a check box or radio filter, in addition to the check box, the filter text can also be clicked. https://prnt.sc/Atdxn2TdtRun

But it would be great if you could add a new option so we can specify that the filter is only activated when the checkbox or radio is clicked. In fact, by enabling this option, the filter will not be activated when users click on the filter text. Rather, they must click on the checkbox or radio.
This is not a specific method. Most of the filters on different websites work in the same way. It would be great if this was added as a new option to both checkbox and radio filters. 

![Screenshot (912)](https://user-images.githubusercontent.com/77163248/202900215-955bc888-021e-4e1b-9a6d-53b5078b7059.jpg)

thank you

**Comment by Lonsdale201** [comment]
As an optional feature it's good, but remember that on mobile, for LTR readers, it's a bad solution, because by default the checkbox and radio are on the left, followed by the label, so it's a strain for the user to stretch their finger until they reach the trigger button with great difficulty. (And count the right handers in the formula, who are in the majority.)  Keep an eye out for such small things if you want to please mobile users (and this is a must, because a higher percentage of them browse from a mobile device than from a laptop). :)

**Comment by 79ho3ein** [comment]
I completely agree with you @Lonsdale201, But it wouldn't be bad if this option were optional.

**Comment by Lonsdale201** [comment]
> I completely agree with you @Lonsdale201, But it wouldn't be bad if this option were optional.

Yes of course. :) 
The reason I wrote it down is that a lot of people don't really know about this kind of thing. What constitutes a grey area on a mobile and how to make it as efficient as possible, because many people focus on the layout and forget about the experience. ^^
